src/appleseed/renderer/modeling/project-builtin/cornellboxproject.cpp

```diff
@@ -1014,6 +1017,43 @@ auto_release_ptr<Project> CornellBoxProjectFactory::create()
         // Attach the camera to the scene.
         scene->cameras().insert(camera);
     }
+    
+    //
+    // Environment
+    //
+
+    // Color called "black_radiance" is created and inserted into the scene.
+    static const float black_radiance[] = { 0.0f, 0.0f, 0.0f };
+    scene->colors().insert(
+        ColorEntityFactory::create(
+            "black_radiance",
+            ParamArray()
+                .insert("color_space", "srgb")
+                .insert("multiplier", "0.5"),
+            ColorValueArray(3, black_radiance)));
+
+    // Environment EDF called "black_edf" is created and inserted it into the scene.
+    scene->environment_edfs().insert(
+        ConstantEnvironmentEDFFactory().create(
+            "black_edf",
+            ParamArray()
+                .insert("radiance", "black_radiance")));
+
+
+    // Environment Shader called "black_shader" is created and insert it into the scene.
+    scene->environment_shaders().insert(
+        EDFEnvironmentShaderFactory().create(
+            "black_shader",
```

Comment:
I would call this "environment_shader".

Reply:
@dictoon, doing so will result in lines like ".insert("environment_shader", "environment_shader"))); "; which I think will result in a bit reduced readability. Wouldn't it be better if it is called like "environmentShader" or so.

Reply:
"environment_black_shader" ? We don't use camel case in appleseed

Reply:
Or `black_environment_shader`?